src/Sylius/Bundle/CoreBundle/Validator/Constraints/ChannelCodeCollectionValidator.php

```diff
@@ -28,7 +28,7 @@ public function __construct(private ChannelRepositoryInterface $channelRepositor
     {
     }
 
-    /** @param null|array<array-key, mixed> $value */
+    /** @param array<array-key, mixed>|null $value */
```

Comment:
```suggestion
    /** @param array<array-key, mixed>|mixed $value */
```

Theoretically in can be anything? (not sure if it's a valid docblock though)

Reply:
I set `/** @param mixed $value */` to conform to the parent class documentation